src/main/java/ch/wisv/areafiftylan/security/Token.java

```diff
@@ -66,13 +68,21 @@ public void setExpiryDate(Date expiryDate) {
         this.expiryDate = expiryDate;
     }
 
-    public void setUsed(boolean used) {
-        this.used = used;
+    public boolean getIsExpireable(){
+        return EXPIRATION != 0;
```

Comment:
This is currently not used in the `isExpired()` method, so it's not working as intended. the `isExpired()` is part of `isValid()` so Tokens with `EXPIRATION = 0` no get expired immediately.